### ISSUE TITLE
Add an ARR scenario to the proxy benchmark matrix

### ISSUE THREAD

**Issue by halter73** [aspnet/Benchmarks#1577]
Currently we have proxy benchmark scenarios defined for YARP, Envoy, NGINX and HAProxy. We should look to add IIS's [Application Request Routing Module (ARR)](https://docs.microsoft.com/en-us/iis/extensions/planning-for-arr/using-the-application-request-routing-module) to this list.

**Comment by halter73** [comment]
So I searched this repo for any existing references to ARR (only after filing this issue ofc 😄), and noticed @sebastienros already added an ARR benchmark app back in February with https://github.com/aspnet/Benchmarks/commit/acfff0cb1c15d57efd4f6455cbdc955a5b2480ed. It doesn't look to be part of any benchmark matrix however.

**Comment by sebastienros** [comment]
That was the goal, compare it with yarp, but haven't found the time to finish it yet.

**Comment by DamianEdwards** [comment]
Closing this out due to age. Feel free to open a new issue if required.
